pages/common/gum.md

```diff
@@ -18,12 +18,12 @@
 
 - Show a spinner while a command is taking place with text alongside:
 
-`gum spin {{[-s|--spinner]}} {{dot|line|minidot|jump|pulse|points|globe|moon|monkey|meter|hamburger}} --title "{{loading...}}" -- {{command}}`
+`gum spin {{[-s|--spinner]}} {{dot|line|minidot|jump|pulse|points|globe|moon|monkey|meter|hamburger}} --title "{{loading...}}" {{command}}`
 
 - Format text to include emojis:
 
 `gum format {{[-t|--type]}} {{emoji}} "{{:smile: :heart: hello}}"`
 
-- Interactively prompt for multi-line text (`<Ctrl d>` to save) and write to `data.txt`:
```

Comment:
Why remove the exit instructions? It's not self-evident

Reply:
I'm assuming they changed the programsince the page was created. If the keybinding are shown, I'm fine with removing the hint.